website/docs/cdktf/cli-reference/commands.mdx

```diff
@@ -380,10 +380,13 @@ $ cdktf synth --help
 **Help Output**
 
 ```
-cdktf synth [OPTIONS]
+cdktf synth [OPTIONS] <stacks..>
 
 Synthesizes Terraform code for the given app in a directory.
 
+Positionals:
+  stacks  Stacks to output when using --json flag
```

Comment:
I think, I'd rather have `--json` take an array here. Else `cdktf synth myStack` would be valid (?) and that might cause confusion because it still touches (i.e. regenerates / changes all files).

Reply:
This is how it worked before, I am open to changing it :)